### PR TITLE
Number of views in blogs added

### PR DESCRIPTION
**Pull request by Rimpi246**
## What is the change?
Added number of views in each blog [For that had to add one more element to the blog document ]

## Related issue?
issue_number#413 is also solved
close: #244 and #413 

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [] Have you made corresponding changes to the documentation?
- [] Your submission doesn't break any existing feature.
- [] Have you tested the code before submission?

## Screenshots or Video:
Add a screenshot or demo video if appropriate.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,5 +28,13 @@
...
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Rimpi246/BlogSite.git"
+  },
+  "bugs": {
+    "url": "https://github.com/Rimpi246/BlogSite/issues"
+  },
+  "homepage": "https://github.com/Rimpi246/BlogSite#readme"
 }
```
Comment (Pranav016) [inline package.json:40]: redundant changes, not required.
restore to previous file

**Review by Pranav016 — CHANGES_REQUESTED** [review]
* resolve conflicts, I'd suggest pull the new changes
* change default tab space in vs code from 2 to 4
* add a demo video or screenshot in PR description

**Comment by Pranav016** [comment]
can you resolve conflicts and add a demo video of your feature ? I can merge it then..

**Comment by Rimpi246** [comment]
> can you resolve conflicts and add a demo video of your feature ? I can merge it then..

Yes sir, I am working on it

**Comment by Pranav016** [comment]
closing for now, reopen if required

**Pull request closed without merging**